### PR TITLE
Ft add monitor code

### PR DESCRIPTION
**Pull request by esirK**
## Description

Adds `monitor_code` in order to know who created a code.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
## Screenshots
Screenshot 2020-09-17 at 13 23 27

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: forms/modelutils.py
Comment (kilemensi) [inline forms/modelutils.py]: Make it a `TextField`... I _think_ the code will be an alphanumeric value,

**Review by kilemensi — APPROVED** [review]
:rocket: @esirK 

--------------------

1. Translations: **fr**: `Code du moniteur`, **es**: `Código del monitor`,
2. Make `Monitor code` the first field in **Basic Information** tab (I've only looked under Newspaper and it's 2nd).

File: forms/modelutils.py
Comment (esirK) [reply forms/modelutils.py]: I think `CharField` will work. `TextField` will produce a `Textarea` which isn't what we want.

**Review by kilemensi — APPROVED** [review]
:rocket:

**Comment by kilemensi** [comment]
Is anything blocking this one @esirK ?

**Comment by esirK** [comment]
> Is anything blocking this one @esirK ?

not really. Since I branched this from the `hot-fix` branch, I wanted to keep the code at https://github.com/CodeForAfrica/gmmp/pull/81 specific to that PR and make it easy to review. Once that's merged I can just merge this to master. @kilemensi

**Pull request merged**
